openapi/components/schemas/Rules/Actions/create-intuit-quickbooks-invoice.yaml

```
@@ -5,10 +5,10 @@ allOf:
     properties:
       customerDisplayName:
         description: |
-          The customer display name in the QuickBooks. Same names will be mapped to a single Quickbooks customer.
+          The customer display name in the QuickBooks. Same names will be mapped to a single QuickBooks customer.
```

nitpick: where possible, we should avoid starting descriptions with "the""

This can make the docs tiresome to read.

I'm sure what is meant by 'same names'

```suggestion
          Customer display name in QuickBooks. Matching names are mapped to a single QuickBooks customer.
```
or
```suggestion
          Customer display name in QuickBooks. Duplicate names are mapped to a single QuickBooks customer.
```

---

Thanks for that!